src/Controls/src/Core/Compatibility/Handlers/Windows/FrameRenderer.cs

```diff
@@ -13,12 +13,33 @@ namespace Microsoft.Maui.Controls.Handlers.Compatibility
 	public class FrameRenderer : ViewRenderer<Frame, WBorder>
 	{
 		public static IPropertyMapper<Frame, FrameRenderer> Mapper
-			= new PropertyMapper<Frame, FrameRenderer>(VisualElementRendererMapper);
+			= new PropertyMapper<Frame, FrameRenderer>(VisualElementRendererMapper)
+			{
+				// https://github.com/dotnet/maui/issues/11880
```

Comment:
Fixing the above issue would most likely make adding these mappers unnecessary. If said fix could be back ported to .NET7 then 👍but if not, then these changes are still relevant. 

I tried to focus these changes purely around `Frame` to make this more back portable.  I'm not aware of other open issues that would be resolved by fixing #111880 currently which is another reason I'm going with the current assumption that we wouldn't fix #11880 for .NET 7,